app/components/LoginForm/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LoginForm = props => {
+  console.log({ props });
```

Comment:
i guess this shouldn't be here, if it's for development purpose you can use the react devtools

Reply:
I removed it.